include/invert_quda.h

```diff
@@ -812,6 +837,37 @@ namespace quda {
     void operator()(ColorSpinorField &out, ColorSpinorField &in);
   };
 
+  class CACGNE : public CACG {
+
+  private:
```

Comment:
clang-format suggested the following change:


```
-  private:
+private:
```

<!-- identifier: clang-format -->
<!-- File: include/invert_quda.h -->